code/_onclick/hud/radial.dm

```diff
@@ -73,17 +81,16 @@ GLOBAL_LIST_EMPTY(radial_menus)
 
 	var/hudfix_method = TRUE //TRUE to change anchor to user, FALSE to shift by py_shift
 	var/py_shift = 0
-	var/entry_animation = TRUE
 
 //If we swap to vis_contens inventory these will need a redo
 /datum/radial_menu/proc/check_screen_border(mob/user)
-	var/atom/movable/AM = anchor
-	if(!istype(AM))
+	if(!istype(anchor))
 		return
+	current_user = user.client
 	var/mob/living/carbon/H
 	if(ishuman(user))
 		H = user
-	if((AM in user.client.screen) || (H && (AM in H.internal_organs)))
+	if((anchor in user.client.screen) || (H && (anchor in H.internal_organs)))
 		if(hudfix_method)
```

Comment:
This seems to always be true. Is anything below this needed besides the anchor = user?